vnext/Microsoft.ReactNative.Cxx/ShareToMac.bat

```diff
@@ -0,0 +1,24 @@
+pushd .
```

Comment:
Please move this file to the scripts folder.

Reply:
This file will be deleted once involved C++ files are shared. It is a temporary tool for me to sync files between two cloned repos. It is OK to put it to the scripts folder?

Reply:
I decide to remove this bat file